docs/migrating-angularJS-tutorial.md

```diff
@@ -1,299 +0,0 @@
----
```

Comment:
I think this tutorial has some value, but it's also outdated. I am thinking we should remove?